src/app/grid/grid-sample-4/grid-sample-4.component.html

```diff
@@ -1,7 +1,50 @@
 <div class="sample-wrapper dark-grid">
-    <igx-grid #grid1 [data]="remoteData | async" (onDataPreLoad)="dataLoading($event)" [height]="'550px'">
-        <igx-column *ngFor="let c of columns" [field]="c.field" [header]="c.header">
+    <igx-grid #grid1 [data]="remoteData | async" [height]="'500px'" [width]="'100%'" [autoGenerate]='false' (onDataPreLoad)="dataLoading($event)">
+        <igx-column [field]="'Category'" [width]="'120px'" [filterable]='true' ></igx-column>
```

Comment:
Do not set filterable columns, filtering works on local level only

Reply:
@dkamburov Ok. Removing it.